src/liquid/components/ld-bg-cells/readme.md

```diff
@@ -158,53 +183,120 @@ A background pattern with the Merck cells as additional visual element.
 
 <style>
   .custom {
-    --ld-bg-cells-position: bottom left;
-    --ld-bg-cells-size: 150%;
+    --ld-bg-cells-layer-translation-x: 26%;
+    --ld-bg-cells-layer-translation-y: -5%;
+    --ld-bg-cells-layer-size: 150%;
     aspect-ratio: 1;
   }
 </style>
 
 <!-- CSS component -->
 
 <div class="ld-bg-cells custom">
-  <div class="ld-bg-cells__pattern" style="--ld-bg-cells-image:url('/dist/build/assets/hexagon-cell.svg')"></div>
+  <div class="ld-bg-cells__layer" style="--ld-bg-cells-image:url('/dist/build/assets/hexagon-cell.svg')"></div>
 </div>
 
 <style>
   .custom {
-    --ld-bg-cells-position: bottom left;
-    --ld-bg-cells-size: 150%;
+    --ld-bg-cells-layer-translation-x: 26%;
+    --ld-bg-cells-layer-translation-y: -5%;
+    --ld-bg-cells-layer-size: 150%;
     aspect-ratio: 1;
   }
 </style>
 {% endexample %}
 
+## Custom Colors
+
+{% example '{ "hasPadding": false, "hasBorder": false }' %}
+
+<ld-bg-cells class="custom-colors"></ld-bg-cells>
+
+<style>
+  .custom-colors {
+    --ld-bg-cells-layer-col: var(--ld-col-rr);
+    --ld-bg-cells-bg-col: var(--ld-col-vy);
+    aspect-ratio: 16/9;
+  }
+</style>
+
+<!-- CSS component -->
+
+<div class="ld-bg-cells custom-colors">
+  <div class="ld-bg-cells__layer" style="--ld-bg-cells-image:url('/dist/build/assets/hexagon-cell.svg')"></div>
+</div>
+
+{% endexample %}
+
+## Three Colors
+
+{% example '{ "hasPadding": false, "hasBorder": false }' %}
+
+<ld-bg-cells class="three-colors"></ld-bg-cells>
+
+<style>
+  .three-colors {
+    --ld-bg-cells-image: url('/dist/build/assets/hexagon-cell.svg');
+    --ld-bg-cells-bg-col: var(--ld-col-vg);
+    --ld-bg-cells-layer-col: var(--ld-col-vy);
```

Comment:
I think we should remove custom color settings from this example and set reasonable defaults in a way that the three layer background pattern version is themable. This should work with the combination of `--ld-thm-primary`, `--ld-thm-secondary` and `--ld-thm-warning` colors.